packages/server/src/main/java/software/uncharted/terarium/hmiserver/utils/rebac/ReBACService.java

```diff
@@ -613,4 +624,9 @@ public List<UUID> lookupResources(final SchemaObject who, final Schema.Permissio
 	public static boolean isServiceUser(final String id) {
 		return API_SERVICE_USER_ID != null && API_SERVICE_USER_ID.equals(id);
 	}
+
+	@Observed(name = "function_profile")
+	public static boolean isAdminServiceUser(final String id) {
+		return ADMIN_API_SERVICE_USER_ID != null && ADMIN_API_SERVICE_USER_ID.equals(id);
```

Comment:
That is what I was afraid of. Do we want to create a test to know if a user is an admin, or set that up properly in Keycloak/ReBac? Of course this will be done in task unrelated to this PR.

Reply:
Fixed